src/applications/vre/28-1900/config/chapters/communication-preferences/communicationPreferences.js

```diff
@@ -91,9 +91,9 @@ export const uiSchema = {
     ),
     'ui:validations': [validateAtLeastOneSelected],
     'ui:options': {
-      // this needed because ObjectField adds the schemaform-block class to this, which creates a huge top margin.
-      classNames: 'vads-u-margin-top--neg2',
-      showFieldLabel: true,
+      // negative margin is needed because ObjectField adds the schemaform-block class to this, which creates a huge top margin.
+      classNames: 'vads-u-margin-top--neg3',
```

Comment:
I think the `row` class name also adds a negative margin... not sure if it's the same as `neg3`